views/pages/GraphPage.jsx

```diff
@@ -156,15 +157,16 @@ const GraphPage = () => {
     let updatedSvgElement = el.current.lastChild
     if (updatedSvgElement) {
       let svgString = new XMLSerializer().serializeToString(updatedSvgElement)
-      let svgUrl =
-        'data:image/svg+xml; charset=utf8, ' + encodeURIComponent(svgString)
 
       const kanvas = canvasRef.current
       kanvas.width = updatedSvgElement.getBBox().width
       kanvas.height = updatedSvgElement.getBBox().height
 
+      let svgUrl =
+        'data:image/svg+xml; charset=utf8, ' + encodeURIComponent(svgString.replace('<svg xmlns="http://www.w3.org/2000/svg" width="100%" height="100%">', `<svg xmlns="http://www.w3.org/2000/svg" width="${kanvas.width}" height="${kanvas.height}">`))
```

Comment:
so using percentage widths doesn't work to fill up the height/width? That's surprising to me. Will this implementation continue to work as the user changes the screen size?

Reply:
Percentage widths work fine in the browser but when we render on a `canvas` it stops working 🤷 . And yes these does work when the screen size changes.